packages/notifier/src/notifier.js

```diff
@@ -12,11 +12,20 @@ import './types.js';
 
 /**
  * @template T
- * @param {ERef<BaseNotifier<T>>} baseNotifierP
- * @returns {AsyncIterable<T> & SharableNotifier}
+ * @typedef {BaseNotifier<T>} ConcreteNotifierInternals
  */
-export const makeNotifier = baseNotifierP => {
-  const asyncIterable = makeAsyncIterableFromNotifier(baseNotifierP);
+
+/**
+ * @template T
+ * @param {ERef<ConcreteNotifierInternals<T>>} concreteP
+ * @returns {AsyncIterable<T> & SharableNotifier<T>}
+ */
+const internalMakeNotifier = concreteP => {
+  const asyncIterable = makeAsyncIterableFromNotifier(concreteP);
+
+  // Cast to preserve sharable opacity.
+  const unknownInternalsP = /** @type {unknown} */ (concreteP);
+  const sharableInternalsP = /** @type {ERef<SharableNotifierInternals<T>>} */ (unknownInternalsP);
```

Comment:
```suggestion
  const sharableInternalsP = 
    /** @type {ERef<SharableNotifierInternals<T>>} */ (unknownInternalsP);
```
This was well over 80 column limit, but I see `yarn lint-fix` doesn't fix it. Why not?

Reply:
Several others too.